runtime/src/current_sandbox.c

```diff
@@ -10,12 +10,16 @@
 #include "sandbox_set_as_complete.h"
 #include "sandbox_set_as_running_user.h"
 #include "sandbox_set_as_running_sys.h"
-#include "sandbox_setup_arguments.h"
 #include "scheduler.h"
 #include "software_interrupt.h"
+#include "wasi_impl.h"
 
 thread_local struct sandbox *worker_thread_current_sandbox = NULL;
 
+// TODO: Propagate arguments from *.json spec file
+const int   dummy_argc   = 1;
+const char *dummy_argv[] = { "Test" };
```

Comment:
That's right! For wasm, it's usually just the executable name because the path info is privileged. We're not respecting that currently, but I don't think it's a big deal. Each sandbox would just think it's named "Test" as is.